conda_recipes/cinema4d-openjd/recipe/activate.sh

```diff
@@ -0,0 +1,13 @@
+export DEADLINE_CLOUD_PYTHONPATH="$CONDA_PREFIX\\Lib\\site-packages"
+export DEADLINE_CINEMA4D_EXE="$CONDA_PREFIX\\c4d\\Commandline.exe"
```

Comment:
This is a property that should be set by the Cinema4D package, not here. That package could choose a different prefix for installation, for example, and this adaptor wouldn't work with it anymore.

Reply:
I agree 100%. I'll update this.